README.md

```diff
@@ -1,19 +1,70 @@
-# Action Name
+# RubyGems Setup GPR Action
```

Comment:
I think it would make this more discoverable and easier to use in the future if we spelt out GPR in the title, and in the name of the github repository too. 
I think I'm dumb, but I can't get my head to remember what GPR means! 
This feels both a silly and an important thing to be commenting on, so sorry. 😄 

I think it might be that I'm expecting it to be GHPR, like (ghcr.io).

Reply:
I have wondered about this as well. Typing it out in full makes for a very cumbersome name for repos, actions and stuff, but I have also struggled to remember the acronym at times. I went with it as GPR is used alot by GH them selves to refer to the service. I think using GHPR would therefore add confusion as not seen that used, even though GHCR is a thing (and yes that is also confusing, thanks GH ;-)

I have tried to expand the acronym in all the docs and descriptions to aid discoverability.

```yaml
  -name: Setup GitHub Package Registry
   uses: fac/rubygems-setup-github-package-registry@v1
```

hmm maybe it is not that bad. Now would be the time to change it, if we are.

Reply:
👆 ! I was wondering what GitHub were shortening it too. I can't find to see anywhere where they're using GPR in their docs still though.

Both ghpr.io and ghr.io are for sale:

Should I grab it?! 😆 
![image](https://user-images.githubusercontent.com/16309691/114692714-c3814a80-9d10-11eb-9701-e046bd45a547.png)

Reply:
I say use GHPR and change the name to longhand. It's something you don't type very often and will read 100000x more often.

Reply:
😂 I'm now the proud owner of ghpr.io.